core/src/VeyonConnection.cpp

```diff
@@ -22,6 +22,10 @@
  *
  */
 
+#ifdef __APPLE__
+	#include <array>
+#endif
+
```

Comment:
If this is required on macOS, don't hesitate to include it independent of the platform to avoid `#ifdef __APPLE__` wherever possible.

Reply:
Clang  on MacOs refuses to build when this is missing, whenever std::array is used somewhere. I think it wouldn't hurt to just include it for every platform, but wasn't sure if this would be appropriate.

Reply:
As of 9d76a8926b6df759fc246d98bd4eb226b0721b1f you shouldn't need it for any source files any longer.